Kerberos.NET/NetStandardExtensions.cs

```diff
@@ -0,0 +1,68 @@
+using System;
```

Comment:
Was this brought over from the dotnet repos or handwritten? If brought over can you stick a copyright similar to the ASN1 bits? Otherwise, nice!

```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

Reply:
Copied out of decompiler and changed into extension methods (in netcoreapp they are part of Stream class).

Reply:
Would it be worthwhile to find it in the original source and bring the methods over verbatim for comments and things? Ideally I'd like to track where things diverge from corefx so they can potentially be cleaned up as necessary in the future.

https://github.com/dotnet/corefx/blob/2858aee575e53a2d5e117d14cb8632f52f65fb70/src/Common/src/CoreLib/System/IO/Stream.cs